core/src/androidTest/java/com/amplifyframework/devmenu/DevMenuMainViewInstrumentationTest.java

```diff
@@ -60,7 +59,7 @@ public void resetView() {
      * Tests that the app is navigated to the environment information
      * screen when the view environment information button is pressed.
      */
-    @Test
+    @Ignore("DeviceFarm does not appear to recognize ignore at the class level.")
```

Comment:
Do we know how/why these tests fail? Can we add that description here, instead of this note?